RELEASE/scripts/autoscend/quests/level_10.ash

```diff
@@ -11,8 +11,8 @@ boolean L10_plantThatBean()
 	string page = visit_url("place.php?whichplace=plains");
 	if(contains_text(page, "place.php?whichplace=beanstalk"))
 	{
-		auto_log_warning("I have no bean but I see a stalk here. Okies. I'm ok with that", "blue");
-		visit_url("place.php?whichplace=beanstalk");
+		auto_log_warning("I see the beanstalk has already been planted. fixing questL10Garbage to step1", "blue");
+		set_property("questL10Garbage", "step1");
```

Comment:
instead of "clicking" on the beanstalk via visit url as a means of correcting mafia tracking (assuming mafia catches the error when you click on it). just advance the quest by a step from "need to plant a bean" to "bean has been planted". (this overall function will only be run if you are at the "need to plant a bean" stage of this tracker)